### PR TITLE
[DM-3867] The OPC UA gateway no longer fails when running in a Docker container that lacks specific 64-bit system functions

### PR DESCRIPTION
**Pull request by bggov**
…

### CODE REVIEW

File: content/change-logs/device-management/cumulocity-10-18-523-0-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-3867
+version: 10.18.523.0
+---
+In certain ARM64-based environments, the OPC UA gateway may fail when attempting to call a specific 64-bit system function that might be missing in some base Docker images. This issue has now been resolved. If this function is missing, the service will attempt to call a default system function before failing.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-523-0-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md:17]:
```suggestion
In certain ARM64-based environments, the OPC UA gateway may have failed when attempting to call a specific 64-bit system function that might be missing in some base Docker images. This issue has now been resolved. If this function is missing, the service will attempt to call a default system function instead.
```

File: content/change-logs/device-management/cumulocity-10-18-523-0-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: 
+title: The OPC UA gateway no longer fails when running in a Docker container that lacks specific 64-bit system functions
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-523-0-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md:3]:
```suggestion
title: OPC UA gateway no longer fails when attempting to call a missing 64-bit system function
```

Comment (Oezge-Akin) [reply content/change-logs/device-management/cumulocity-10-18-523-0-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md:3]: Shouldn't it say directly ARM 64-bit system in the title?

Comment (bggov) [reply content/change-logs/device-management/cumulocity-10-18-523-0-OPC-UA-Gateway-run-on-arm64-without-speciffic-64bist-system-libs.md:3]: The issue might happen not only on ARM 64 but any 64-bit system where the fallocate64() is missing. As an example you can have a 32-bit installation/image running on any 64-bit system. In this case the 64-bit system functions are missing.

**Review by Oezge-Akin — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**